tinygrad/runtime/ops_gpu.py

```diff
@@ -87,7 +86,7 @@ def __call__(self, global_size, local_size, *bufs, wait=False) -> Optional[float
 
 class CLCodegen(CStyleCodegen):
   lang = CStyleLanguage(
-    kernel_prefix = "#define int64 long\n__kernel", buffer_prefix = "__global ", smem_prefix = "__local ",
+    kernel_prefix = "#ifdef cl_khr_fp64\n#pragma OPENCL EXTENSION cl_khr_fp64 : enable\n#elif defined(cl_amd_fp64)\n#pragma OPENCL EXTENSION cl_amd_fp64 : enable\n#endif\n#define int64 long\n__kernel", buffer_prefix = "__global ", smem_prefix = "__local ",
```

Comment:
Want to do something similar to half_prekernel so this isn't in all the kernels?

Reply:
misread this, ignore the last commit